wpilibj/src/main/java/edu/wpi/first/wpilibj/SensorBase.java

```diff
@@ -17,70 +17,67 @@
 import edu.wpi.first.wpilibj.hal.SolenoidJNI;
 
 /**
- * Base class for all sensors. Stores most recent status information as well as containing utility
- * functions for checking channels and error processing.
+ * Stores most recent status information as well as containing utility functions for checking
+ * channels and error processing.
  */
-public abstract class SensorBase extends SendableBase {
+public class SensorBase {
   /**
    * Ticks per microsecond.
    */
-  public static final int kSystemClockTicksPerMicrosecond =
-      ConstantsJNI.getSystemClockTicksPerMicrosecond();
+  public static int kSystemClockTicksPerMicrosecond
+      = ConstantsJNI.getSystemClockTicksPerMicrosecond();
+
   /**
    * Number of digital channels per roboRIO.
    */
-  public static final int kDigitalChannels = PortsJNI.getNumDigitalChannels();
+  public static int kDigitalChannels = PortsJNI.getNumDigitalChannels();
+
   /**
    * Number of analog input channels per roboRIO.
    */
-  public static final int kAnalogInputChannels = PortsJNI.getNumAnalogInputs();
+  public static int kAnalogInputChannels = PortsJNI.getNumAnalogInputs();
+
   /**
    * Number of analog output channels per roboRIO.
    */
-  public static final int kAnalogOutputChannels = PortsJNI.getNumAnalogOutputs();
+  public static int kAnalogOutputChannels = PortsJNI.getNumAnalogOutputs();
+
   /**
    * Number of solenoid channels per module.
    */
-  public static final int kSolenoidChannels = PortsJNI.getNumSolenoidChannels();
+  public static int kSolenoidChannels = PortsJNI.getNumSolenoidChannels();
+
   /**
    * Number of PWM channels per roboRIO.
    */
-  public static final int kPwmChannels = PortsJNI.getNumPWMChannels();
+  public static int kPwmChannels = PortsJNI.getNumPWMChannels();
+
   /**
    * Number of relay channels per roboRIO.
    */
-  public static final int kRelayChannels = PortsJNI.getNumRelayHeaders();
+  public static int kRelayChannels = PortsJNI.getNumRelayHeaders();
+
   /**
    * Number of power distribution channels per PDP.
    */
-  public static final int kPDPChannels = PortsJNI.getNumPDPChannels();
+  public static int kPDPChannels = PortsJNI.getNumPDPChannels();
+
   /**
    * Number of power distribution modules per PDP.
    */
-  public static final int kPDPModules = PortsJNI.getNumPDPModules();
-  /**
-   * Number of PCM Modules.
-   */
-  public static final int kPCMModules = PortsJNI.getNumPCMModules();
-
-  private static int m_defaultSolenoidModule = 0;
+  public static int kPDPModules = PortsJNI.getNumPDPModules();
```

Comment:
Why can't these be final anymore?

Reply:
This used to be part of the MotorSafety PR and it turned SensorBase into an interface. interface member variables are `public static final` by default. I forgot to add the keywords back in when I made this a class again.